src/scripts/components/ResourceIntro.jsx

```diff
@@ -135,8 +144,10 @@ class ResourceIntro extends React.Component {
 ResourceIntro.propTypes = {
   avatar: PropTypes.shape({
```

Comment:
consider passing `avatar` and `multiAvatar` as their own prop shapes so that we're not mixing avatar props with multiavatar props.

Reply:
done!